pandas/core/frame.py

```diff
@@ -22,6 +22,7 @@
 from numpy import nan as NA
 import numpy as np
 import numpy.ma as ma
+import IPython
```

Comment:
this is not a required dependency, so do this in the actual function. surround with a ``try/except``. if you cannot figure out the version (e.g. ipython is not installed), then use the new style is ok.